### PR TITLE
fix convert_forum_subscriptions in Merge_fluxBB

### PR DESCRIPTION
**Pull request by kichawa**
fix problem with forum_subscriptions table during merge from FluxBB to FluxBB

### CODE REVIEW

File: tests/ConverterTest.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 if (is_dir(CONV_ROOT.'fluxbb/'))
 	define('PUN_ROOT', CONV_ROOT.'fluxbb/');
 
-require_once 'PHPUnit/Framework/TestCase.php';
+require_once 'PHPUnit/Autoload.php';
```
Comment (franzliedke) [inline tests/ConverterTest.php]: Why this change? This [broke our build](https://travis-ci.org/fluxbb/converter/builds/77185502).

**Comment by franzliedke** [comment]
Thank you! Please revert the change to the test file, so I can merge. :)

File: tests/ConverterTest.php (hunk 1 above)
Comment (kichawa) [inline tests/ConverterTest.php]: The build was broken even without this change. I thought this change makes pass the test.

Comment (franzliedke) [inline tests/ConverterTest.php]: Ah yes, sorry about that. I'll take a look.

**Comment by kichawa** [comment]
Done.

**Comment by franzliedke** [comment]
Thanks a lot. I fixed the tests.

**Pull request merged**